utils/entrypoint.sh

```diff
@@ -11,14 +11,4 @@ runner:x:`id -u`:`id -g`:,,,:/home/runner:/bin/bash
 EOF
 fi
 
-if [[ -n "${LAUNCHED_BY_RUNNER}" ]]; then
-    RUNNER_CALLBACKS=$(python3 -c "import ansible_runner.callbacks; print(ansible_runner.callbacks.__file__)")
-
-    # TODO: respect user callback settings via
-    # env ANSIBLE_CALLBACK_PLUGINS or ansible.cfg
-    export ANSIBLE_CALLBACK_PLUGINS="$(dirname $RUNNER_CALLBACKS)"
-
-    export ANSIBLE_STDOUT_CALLBACK=awx_display
-fi
-
```

Comment:
@shanemcd this entrypoint stuff exists in order to ascertain the location of the `ansible-runner` install and put them in the env vars needed for Ansible. This proposal makes that unnecessary by putting the collection in a location expected by Ansible, which is `/usr/share/ansible/collections`, which is the same as `ansible-builder`.

It's not just `ANSIBLE_CALLBACK_PLUGINS`, it's also `PYTHONPATH` that has historically been set and used this location. However, I think that may been erroneous because the system path change was done in the callback python code.